stdlib/src/builtin/math.mojo

```diff
@@ -262,7 +262,17 @@ trait Powable:
     Types that conform to `Powable` will work with the builtin `pow` function,
     which will return the same type as the inputs.
 
-    TODO: add example
+    For example:
+    ```mojo
+    @value
+    struct Rational(Powable):
+        var numerator: Float64
+        var denominator: Float64
+
+        fn __pow__(self,exp: Self)  -> Self:
+            var exp_value = exp.numerator / exp.denominator
+            return Self(pow(self.numerator,exp_value), pow(self.denominator,exp_value))
+    ```
```

Comment:
Might be worth giving an example of how this enables the exponentiation operator, `**`. This would make it consistent with the others :) Reference here https://github.com/modularml/mojo/blob/1f9ccc7cc07fa1d70c02ba6df3bc7b92f54c85fb/stdlib/src/builtin/value.mojo#L19-L48 

Also, a minor formatting nit: it would be clearer to space the parameters.

Reply:
Thanks for the feedback! Should all be addressed.

Reply:
Hi was also wondering, this is my first time contributing here. Should I mark this as resolved or do reviewers mark? Thanks!

Reply:
Looks good to me, thanks! We don't have anything formal yet around "who resolves the conversations" on the PR.  I've synced this PR internally and it'll land here soon.